students/Herman_Chloe.md

```diff
@@ -0,0 +1,6 @@
+An empirical test of the role of small-scale transmission in large-scale disease dynamicsThe American Naturalist
```

Comment:
It is food Chloe!
You can use headings by starting lines with # (level 1):
# Heading 1
Or ## (level 2):
## Heading 2
Or ### (level 3):
### Heading 3